content/hi/docs/reference/glossary/toleration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Toleration
```

Comment:
Optionally, use Hindi  here; you can then use `aka` to also show the English term.